lints/package.json

```diff
@@ -31,6 +32,7 @@
     "redux": "3.6.0",
     "redux-form": "6.1.0",
     "redux-thunk": "2.1.0",
+    "serialize-to-js": "^0.5.0",
```

Comment:
It's a library with a known vulnerability that I'm using to test the CircleCI job.